test/functional/lua/loop_spec.lua

```diff
@@ -52,8 +53,10 @@ describe('vim.loop', function()
 
     eq(0, meths.get_var('coroutine_cnt'))
     exec_lua(code)
-    sleep(50)
-    eq(2, meths.get_var('coroutine_cnt'))
+    retry(2, nil, function()
+      sleep(50)
+      eq(2, meths.get_var('coroutine_cnt'))
+    end)
```

Comment:
Could maybe retry more often?  OTOH all those issues might also indicate some problem with libuv on osx in general?